### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by nathanshammah**
Update readme as discussed in recent chat with @quantshah and @araza6. Add information on:
- installation
-requirements
- development
- documentation 
- information on the library

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,5 +4,40 @@ A Python library to integrate automatic differentiation tools such as Jax with Q
...
+
+``numpy scipy matplotlib cython pytest qutip jax``
+
+
+## About
+
+``qgrad`` is a library that implements Hamiltonian learning in the context of quantum physics-based optimization tasks.
+ ``qgrad`` reproduces essential [QuTiP](http://qutip.org/) functions to reduce the friction for existing QuTiP users.
+
+While many quantum libraries focus on quantum circuit learning, ``qgrad`` focuses on Hamiltonian learning.
```
Comment (araza6) [inline README.md:34]: @quantshah suggested removing this comparision. I have removed this from the home docs as well

Comment (araza6) [reply README.md:34]: I should have been clearer, I was talking about line 34

Comment (nathanshammah) [inline README.md:34]:
```suggestion
```

**Comment by nathanshammah** [comment]
Thanks @araza6, I deleted that line and merge.

**Pull request merged**